docs/securing_installation.md

```diff
@@ -0,0 +1,108 @@
+
+## Who Needs Security Configurations?
+
+For the following types of clusters we strongly recommend that you apply the proper security configurations to Helm and Tiller to ensure the safety of the cluster, the data in it, and the network to which it is connected.
+
+- Clusters that are exposed to uncontrolled network environments
+- Clusters that are for many people to use -- _multitenant_ clusters -- as a shared environment
+- Clusters that have access to or use high-value data or networks of any type
+
+Often, environments like these are referred to as _production grade_ or _production quality_ because the damage done to any company by misuse of the cluster can be profound for either customers, the company itself, or both. Once the risk of damage becomes high enough, you need to ensure the integrity of your cluster no matter what the actual risk. 
+
+To configure your installation properly for your environment, you must:
+
+- Understand the security context of your cluster
+- Choose the Best Practices you should apply to your helm installation
+
+The following assumes you have a Kubernetes configuration file (a _kubeconfig_ file) or one was given to you to access a cluster. 
+
+## Understanding the Security Context of your Cluster
+
+`helm init` installs Tiller into the cluster in the `default` namespace and without any RBAC rules applied. This is appropriate for local development and other private scenarios because it enables you to be productive immediately. It also enables you to continue running Helm with existing Kubernetes clusters that do not have role-based access control (RBAC) support until you can move your workloads to a more recent Kubernetes version.
+
+There are four main areas to consider when securing a tiller installation:
+
+1. Role-based access control, or RBAC
+2. Tiller's gRPC endpoint and its usage by Helm
+3. Tiller release information
+4. Helm charts
+
+### RBAC
+
+Recent versions of Kubernetes employ a [role-based access control (or RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) system as do modern operating systems to help mitigate the damage that can done if credentials are misused or bugs exist. Even in where an identity is hijacked, the identity has only so many permissions to a controlled space. This effectively adds a layer of security to limit the scope of any attack with that identity. 
+
+Helm and Tiller are designed to install, remove, and modify logical applications that can contain many service interacting together. As a result, often its usefulness involves cluster-wide operations, which in a multitenant cluster means that great care must be taken with access to a cluster-wide Tiller installation to prevent improper activity. 
```

Comment:
typo: "... can contain many *services* interacting together ..."

Reply:
Can I pay you to proofread this again? Thank you.